packages/telegraf/extra/telegraf.ps1

```diff
@@ -11,4 +10,22 @@ $local_ip = (Find-NetRoute -RemoteIPAddress $IPAddress | Select-Object  Ipv4Addr
 $env:DCOS_NODE_PRIVATE_IP=$local_ip;
 
 & $pkg_inst_dpath\bin\telegraf.exe --config $dcos_conf_dpath\telegraf\telegraf.conf --service install
+
+# set the Service Recovery options to "Restart the Service" every time the service crashes
+function Set-ServiceRecovery{
+    [alias('Set-Recovery')]
+    param
+    (
+        [string] [Parameter(Mandatory=$true)] $ServiceDisplayName,
+        [string] $action = "restart", 
+        [int] $time =  30000, # in miliseconds
+        [int] $resetCounter = 4000 # in seconds
```

Comment:
Just to confirm, the first number is 30 seconds and the second number is over an hour?

Reply:
Yes, $resetCounter specifies the length of the period (in seconds) with no failures after which the failure count should be reset to 0